json_to_csv.py

```diff
@@ -0,0 +1,46 @@
+import os
+import json
+import csv
+
+
+def fun(in_dir,out_dir,**d):
+
+    os.chdir("C:/Users/anurag/Downloads/darkflow-master/darkflow-master") #change directory to that path where object detector YOLO v2 resides.

```

Comment:
Avoid any hardcoding, these should clearly be input params. Indeed, your code should work relative to the project directory instead of full paths.

Reply:
ok